src/components/HistoryTables/MessageHistory/Detail.tsx

```diff
@@ -29,16 +26,13 @@ import {
 import { useMessage } from '../useAllMessages'
 import { useUnformattedDateTime } from './useAge'
 import { useMethodName } from './useMethodName'
-import Card from '../../Card'
+import { Lines, Line, InfoBox, StandardBox } from '../../Layout'
 
 // add RelativeTime plugin to Day.js
 dayjs.extend(relativeTime.default)
 
-const SeeMore = styled(P).attrs(() => ({
-  color: 'core.primary',
-  role: 'button'
-}))`
-  cursor: pointer;
+const SpanGray = styled.span`
+  color: var(--gray-medium);
```

Comment:
Was wondering is we should make this something generic, but it only lives here for now

Reply:
Think it's fine for now!